src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7190,6 +7182,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ImplicitlyTypedDefaultParameter" xml:space="preserve">
     <value>Implicitly typed lambda parameter '{0}' cannot have a default value.</value>
   </data>
+  <data name="ERR_OptionalParamValueMismatch" xml:space="preserve">
+    <value>Parameter {0} has default value '{1:10}' in lambda and '{2:10}' in the target delegate type.</value>
```

Comment:
>:10 [](http://example.com/codeflow?start=46&length=3)

What does the `:10` format do?

Reply:
It limits the length when a  `ConstantValue` is converted to string, to void serializing huge constant values